ContainerImagesAndGithubActions.md

```diff
@@ -0,0 +1,122 @@
+document.
+
+The ACA-Py images built by this project are tagged to indicate which of the
+above variants it is. Other tags may also be generated for use by developers.
+
+Below is a table of all generated images and their tags:
+
+Tag                     | Variant  | Example                  | Description                                                                                     |
+------------------------|----------|--------------------------|-------------------------------------------------------------------------------------------------|
+py3.6-X.Y.Z             | Standard | py3.6-0.7.4              | Standard image variant built on Python 3.6 for ACA-Py version X.Y.Z                             |
+py3.7-X.Y.Z             | Standard | py3.7-0.7.4              | Standard image variant built on Python 3.7 for ACA-Py version X.Y.Z                             |
+py3.8-X.Y.Z             | Standard | py3.8-0.7.4              | Standard image variant built on Python 3.8 for ACA-Py version X.Y.Z                             |
+py3.9-X.Y.Z             | Standard | py3.9-0.7.4              | Standard image variant built on Python 3.9 for ACA-Py version X.Y.Z                             |
+py3.10-X.Y.Z            | Standard | py3.10-0.7.4             | Standard image variant built on Python 3.10 for ACA-Py version X.Y.Z                            |
+py3.7-indy-A.B.C-X.Y.Z  | Indy     | py3.7-indy-1.16.0-0.7.4  | Standard image variant built on Python 3.7 for ACA-Py version X.Y.Z and Indy SDK Version A.B.C  |
+py3.8-indy-A.B.C-X.Y.Z  | Indy     | py3.8-indy-1.16.0-0.7.4  | Standard image variant built on Python 3.8 for ACA-Py version X.Y.Z and Indy SDK Version A.B.C  |
+py3.9-indy-A.B.C-X.Y.Z  | Indy     | py3.9-indy-1.16.0-0.7.4  | Standard image variant built on Python 3.9 for ACA-Py version X.Y.Z and Indy SDK Version A.B.C  |
+py3.10-indy-A.B.C-X.Y.Z | Indy     | py3.10-indy-1.16.0-0.7.4 | Standard image variant built on Python 3.10 for ACA-Py version X.Y.Z and Indy SDK Version A.B.C |
+
+### Key Image Differences
+
+There are several key differences that should be noted between the two image
+variants and between the BC Gov ACA-Py images.
+
+- Standard Image
+    - Based on slim variant of Debian
+    - Does **NOT** include `libindy`
+    - Default user is `aries`
+    - Uses container's system python environment rather than `pyenv`
+    - Askar and Indy Shared libraries are installed through pip from
+      pre-compiled binaries included in the python wrappers.
+    - Built from repo contents
+- Indy Image
+    - Based on slim variant of Debian
+    - Based on `indy-python`
+    - Includes `libindy` but does **NOT** include the Indy CLI
+    - Default user is `indy`
+    - Based on `indy-python`
+    - Uses container's system python environment rather than `pyenv`
+    - Askar and Indy Shared libraries are installed through pip from
+      pre-compiled binaries included in the python wrappers
+    - Built from repo contents
```

Comment:
This works for release builds only because the images are built at the same time, and therefore with the same dependencies, as the package that is published to PyPi.